src/components/Popups/PopupDrawingToolbar.scss

```diff
@@ -1,5 +1,40 @@
 @import '~box-ui-elements/es/styles/variables';
 
+@mixin PopupDrawingToolbarButton {
```

Comment:
Mixins are this level are global, like variables. We should prefix with `ba-`, probably.